.eslintrc

```diff
@@ -3,34 +3,49 @@
     "es6": true,
     "browser": true
   },
-  "extends": [
-    "pubsweet"
-  ],
+  "settings": {
+    "import/resolver": {
+      "webpack": {
+        "config": "./webpack/webpack.development.config.js"
+      }
+    }
+  },
+  "extends": ["pubsweet"],
   "parser": "babel-eslint",
   "rules": {
     "global-require": 0,
-    "class-methods-use-this": ["error", {
-       "exceptMethods": ["storage",
-        "render",
-        "componentDidMount",
-        "shouldComponentUpdate"]
-     }],
+    "import/no-unresolved": "off",
```

Comment:
Is there any way to whitelist this rule? I feel we'll be losing some useful checks by disabling it globally just to allow aliases for the ui components. Maybe we can look into creating `package.json` files for the ui molecules similar to how we're using the `xpub-server` and `xpub-model` in the server?

Reply:
Fair point ... I did scratch my head over this when reviewing ... glad you picked it up @Cooryd

Reply:
I think this actually can be whitelisted, it was initially put in as an attempt to achieve the changes we wanted but then we found another way by specifying the actual resolver.

Reply:
Creating a pr to fix this.